ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -677,3 +677,30 @@ def verify_storage_clients_page(self):
             page_name="client_onboarding_token_page",
         )
         return storage_client_obj
+
+    def odf_storagesystems_consumption_trend(self):
+        """
+        Function to verify changes and validate elements on ODF storage consumption trend for ODF 4.17
+        This will navigate through below order
+        DataFoundation>>Storage>>storagecluster_storagesystem_details>>Block and File page
+        Further it looks for the Consumption trend card
+
+        Returns:
+            tuple = tpl_of_days_and_avg  ex: (Estimated days, Average)
+
+        """
+        self.odf_console_plugin_check()
+        storage_systems_page = (
+            PageNavigator().nav_odf_default_page().nav_storage_systems_tab()
+        )
+        storage_system_details = (
+            storage_systems_page.nav_storagecluster_storagesystem_details()
+        )
+        storage_system_details.nav_details_overview()
+        storage_system_details.nav_details_object()
+
+        if not config.ENV_DATA["mcg_only_deployment"]:
```

Comment:
pls add else with logger.error to notify test eng why we can't fetch data something like
"no data available for MCG only deployments"

Reply:
done